sdk/src/main/java/io/opentelemetry/sdk/trace/export/SpanExporter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace.export;
+
+import io.opentelemetry.proto.trace.v1.Span;
+import io.opentelemetry.sdk.trace.TracerSdk;
+import java.util.List;
+
+/**
+ * An interface that allows different tracing services to export recorded data for sampled spans in
+ * their own format.
+ *
+ * <p>To export data this MUST be register to the {@code TracerSdk} using a {@link
+ * SimpleSampledSpansProcessor} or a {@code BatchSampledSpans}.
+ */
+// TODO: Change {@code BatchSampledSpansExporter} to {@link BatchSampledSpansExporter} when the
+//  class is available.
+public interface SpanExporter {
+
+  /**
+   * Called to export sampled {@code Span}s.
+   *
+   * @param spans the list of sampled Spans to be exported.
+   */
+  void export(List<Span> spans);
```

Comment:
Why not a single span? Is supposed the span processor batch spans? That interface reports only a single object.

Reply:
(To me it sounds like the `Simple` processor is a kind of toy to show how things work - in a more advanced scenario, a not-so-simple processor ought to buffer them instead)

Reply:
Correct, we will have a batch version that will be more complex.

Reply:
Also may be used in environments where batching may be a problem (like cloud functions, appengine).